Library/Config.ts

```diff
@@ -53,9 +58,21 @@ class Config {
     /** Host name for quickpulse service */
     private _quickPulseHost: string;
 
-    constructor(instrumentationKey?: string) {
-        this.instrumentationKey = instrumentationKey || Config._getInstrumentationKey();
-        this.endpointUrl = `${this.endpointBase}/v2/track`;
+
+    constructor(instrumentationKey?: string, connectionStringCode?: string) {
```

Comment:
How does `connectionStringCode` get populated? I don't see a change to the `setup` method to pass it.

How will `setup` know which field to fill?

Reply:
This constructor is incorrect right now. I am going to combine functionalities into the 1st arg. `setup` will behave the same